core/modules/moduleconf.py

```diff
@@ -30,6 +46,12 @@ class ModuleConf(List):
     kindName = "viur-module-conf"
     accessRights = ["edit"]
 
+    def adminInfo(self):
+        admin_info = super().adminInfo()
+        admin_info["icon"] = "icon-settings"
+        admin_info["display"] = "hidden"
```

Comment:
If the module is hidden, how could it be edited?

Reply:
It can be edited when it's subclassed and the adminInfo is corrected. I don't want to have the ModuleConf-module automatically available and visible in all existing projects now.

Reply:
Ok, proposal: Can we provde the adminInfo via config? 

So the default would be 
```python
conf["viur.module.conf.admin_info"] = {
    "name": "Module Conf",
    "icon": "icon-settings",
    "display": "hidden"
}
```
but it can easily modified project-specific.

Reply:
4ae1bc5 is my proposal on this @sveneberth